test/unit/lib/plugins/aws/package/compile/events/event-bridge/index.test.js

```diff
@@ -564,6 +590,11 @@ describe('EventBridgeEvents', () => {
         expect(deadLetterConfigRuleTarget.DeadLetterConfig).to.have.property('Arn');
       });
 
+      it('should correctly set event name when set', () => {
+        const customNameTarget = getRuleResourceEndingWith(cfResources, 'test').Properties;
```

Comment:
All new tests should be configured purely with `runServerlss` util. See bottom of this test file, I believe you can add there custom name case.

Also check: https://github.com/serverless/serverless/tree/main/test#unit-tests

Reply:
@medikoo tests updated; let me know if there's still any issues